pkg/operator/kube_cloud_config/aws_test.go

```diff
@@ -0,0 +1,353 @@
+package kube_cloud_config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
```

Comment:
nit: no spaces.  gofmt doesn't auto-do it and we decided the separate groups didn't buy us anything.